config/wdio/visualRegressionConf.js

```diff
@@ -0,0 +1,86 @@
+const path = require('path');
+const LocalCompare = require('wdio-visual-regression-service/compare').LocalCompare;
+const viewports = require('./services.default-config').terraViewports;
+
+const screenshotSetup = {
+  diffDir: 'diff',
+  referenceDir: 'reference',
+  screenshotDir: 'latest',
+};
+
+const testIdRegex = /\[([^)]+)\]/;
+
+function createTestName(fullName) {
+  const matches = testIdRegex.exec(fullName);
+
+  // If test ID is provided, use the ID for a shorter test name, otherwise use the full name
+  let name = matches ? matches[1] : fullName.trim();
+
+  // Remove white space
+  name = name.replace(/[\s+.]/g, '_');
+
+  // Remove windows reserved characters. See: https://msdn.microsoft.com/en-us/library/windows/desktop/aa365247(v=vs.85).aspx#naming_conventions
+  // eslint-disable-next-line no-useless-escape
+  name = name.replace(/[\?\<\>\/\\|\*\"\:\+\.]/g, '-');
+
+  return name;
+}
+
+function getScreenshotName(context) {
+  const parentName = createTestName(context.test.parent);
+  const testName = createTestName(context.test.title);
+
+  return `${parentName}[${testName}].png`;
+}
+
+function getFormFactor(context) {
+  let formFactor = global.browser.options.formFactor;
+
+  if (!formFactor) {
+    const browserWidth = context.meta.viewport.width;
+    const viewportSizes = Object.keys(viewports);
+    for (let i = 0; i < viewportSizes.length; i += 1) {
+      const viewport = viewportSizes[i];
+      if (browserWidth <= viewports[viewport].width) {
```

Comment:
We should probably either add a fallback incase the browserWidth is greater than our biggest viewport, in which case `formFactor` would be undefined, or explicitly throw an error.

Reply:
If a viewport is larger than a terra defined viewport, it will be returned as `undefined` and the form dir will then be named `undefined`. This will not break tests. 

I have two thoughts with this,

1. we either check and set the fallback to `enormous` 
2. we do not provide a fallback because this screensize is larger than a terra-defined viewport.

For now, I will update to fallback to `enormous` as this is verifies that at an overlay large viewport, the page/element/etc renders as expected. However, I what are other's opinions on providing this fallback?

@mjhenkes @bjankord @ryanthemanuel @JakeLaCombe